web/models.py

```diff
@@ -222,3 +225,55 @@ def __unicode__(self):
 
     def __str__(self):
         return unicode(self).encode('utf-8')
+
+
+class Meet(models.Model):
+    MEET_TYPE_A = 1
+    MEET_TYPE_B = 2
+    MEET_TYPE_OTHER = 3
+    MEET_TYPE_CHOICES = (
+        (MEET_TYPE_A, 'A'),
+        (MEET_TYPE_B, 'B'),
+        (MEET_TYPE_OTHER, 'Other'),
+    )
+
+    meet_type = models.PositiveSmallIntegerField(choices=MEET_TYPE_CHOICES)
+    name = models.CharField(max_length=200)
+    date = models.DateField()
+
+    def __unicode__(self):
+        return (self.name + " " +
+                dict(self.MEET_TYPE_CHOICES).get(self.meet_type) +
+                " Meet on " + self.date.strftime('%Y-%m-%d'))
+
+    def __str__(self):
+        return unicode(self).encode('utf-8')
+
+
+class MeetDocument(models.Model):
```

Comment:
I don't think this class gets the relationship between meets and documents correctly. The foreign key association doesn't enforce the relationship that **each meet has (optionally) exactly one program and result**. The solution I think is to have two optional `FileField`s, program and result in the `Meet` model.